09-mapping.Rmd

```diff
@@ -109,9 +123,9 @@ map_nz = tm_shape(nz) + tm_polygons()
 class(map_nz)
 ```
 
-`map_nz` can be plotted later, for example by adding additional layers (as we'll below) or simply running `map_nz` in the console, which is equivalent to `print(map_nz)`.
+`map_nz` can be plotted later, for example by adding additional layers (as shown below) or simply running `map_nz` in the console, which is equivalent to `print(map_nz)`.
```

Comment:
Nice remove of unnecessary use of the [plural possessive pronoun](https://www.thefreedictionary.com/We+(word)), thanks for this @Jannes-m, I've been caught red-handed (check the blame)!